styles/styles.css

```diff
@@ -4,9 +4,9 @@
     font-weight: normal;
     src: url("fonts/ionicons.eot?v=2.0.0");
     src: url("fonts/ionicons.eot?v=2.0.0#iefix") format("embedded-opentype"),
-         url("fonts/ionicons.ttf?v=2.0.0") format("truetype"),
-         url("fonts/ionicons.woff?v=2.0.0") format("woff"),
-         url("fonts/ionicons.svg?v=2.0.0#Ionicons") format("svg");
+        url("fonts/ionicons.ttf?v=2.0.0") format("truetype"),
+        url("fonts/ionicons.woff?v=2.0.0") format("woff"),
+        url("fonts/ionicons.svg?v=2.0.0#Ionicons") format("svg");
```

Comment:
Please revert this change.

Reply:
Sorry. I use automatic indentation and I did not realize. Fixed.